server/form-pages/utils/index.ts

```diff
@@ -32,3 +32,18 @@ export const getTask = <T>(task: T) => {
     pages: taskPages,
   }
 }
+
+export const getSection = <T>(section: T) => {
```

Comment:
My preference would have been for this to be in its own commit, as it gets a bit lost among the other changes